### DISCUSSION TITLE
Add Desktop notification support

### DISCUSSION THREAD

**Discussion by Djiit** [containrrr/shoutrrr#352] [Q&A] [answered]
is it planned to add Desktop Notification support ?

**Comment by piksel** [comment, marked as answer]
What OS are you referring to? You can use Gotify, Ntfy, Pushover or any of the other services that have native clients. Communicating directly with the OS is not possible for Windows and probably not for macOS either. It depends on what kind of environment the consumer application runs in, of course. Right now it's mostly targeted towards containers (where the above apply).
